### ISSUE TITLE
Proxy Blade Zero (313980): Black Screen issue 

### ISSUE THREAD

**Issue by waspennator** [FNA-XNA/FNA#491]
Bringing this up here since the proton github discussion suggests it's an FNA problem. Game seems to launch to a black screen visually and I cannot see anything. The sound works fine and I can hear menu noises if I move around or press start. Guessing this might have something to do with it.

[steam-313980.log.txt](https://github.com/user-attachments/files/16488045/steam-313980.log.txt)

00000130:] EXCEPTION handling: System.NotImplementedException: Unhandled render state! 165

Tested on a 1 TB OLED Steam deck LE with proton Experimental

**Comment by roys-coding** [comment]
Any comments on the matter? Did you figure out if it was an FNA problem?

**Comment by waspennator** [comment]
The problem is still occurring, I'm still pretty sure it's an FNA problem considering this game uses XNA and that specific error message is related to FNA

**Comment by flibitijibibo** [comment]
SDL3 still has my attention so it'll be a while, I could maybe fast-track this as part of making a native version though

**Comment by waspennator** [comment]
Game has regressed further, yeets this up after a white screen then crashes.

[steam-313980.log](https://github.com/user-attachments/files/28137229/steam-313980.log)

<img width="1280" height="800" alt="Image" src="https://github.com/user-attachments/assets/c8131a08-6404-435b-b502-208a78fd9737" />

**Comment by 7aGiven** [comment]
Can it run on Windows? If it can run on Windows, why you blame FNA for the issue?

**Comment by waspennator** [comment]
> Can it run on Windows? If it can run on Windows, why you blame FNA for the issue?

Because im trying to run this on my steam deck which isn't working and the proton log and the image are saying its an xna/fna issue. I have run this on windows many years before and know it works there with xna

**Comment by 7aGiven** [comment]
Yeah. I think that if Game works with FNA on Windows, it mean Proton or wine issue. Not a FNA issue.

**Comment by waspennator** [comment]
I have not tried this game with fna on windows, only xna, dont think I knew fna existed when I last played it on Windows long ago
